CMakeLists.txt

```diff
@@ -1,30 +1,43 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(lms1xx)
 
+add_compile_options(-std=c++11)
+
 # Build ROS-independent library.
 find_package(console_bridge REQUIRED)
-include_directories(include ${console_bridge_INCLUDE_DIRS})
+include_directories(include SYSTEM ${console_bridge_INCLUDE_DIRS})
```

Comment:
@ipa-bnm see https://github.com/ipa320/LMS1xx/pull/2#pullrequestreview-37821888 This is not a system dependency...